### ISSUE TITLE
Barcode Layout

### ISSUE THREAD

**Issue by data2base** [opensourcepos/opensourcepos#2588]
### Background information

__IMPORTANT: If you choose to ignore this issue report template, your issue will be closed as we cannot help without the requested information.__

Please make sure you tick (add an x between the square brackets with no spaces) the following check boxes:

- [] Reporting an issue of an unmodified OSPOS installation
- [] Checked [open and closed issues](https://github.com/opensourcepos/opensourcepos/issues?utf8=%E2%9C%93&q=is%3Aissue) and no similar issue was already reported (please make sure you searched!)
- [] Read [README](https://github.com/opensourcepos/opensourcepos/blob/master/README.md), [WHATS_NEW](https://github.com/opensourcepos/opensourcepos/blob/master/WHATS_NEW.txt), [INSTALL.md](https://github.com/opensourcepos/opensourcepos/blob/master/INSTALL.md) and [UPGRADE](https://github.com/opensourcepos/opensourcepos/blob/master/UPGRADE.txt)
- [] Read the [FAQ](https://github.com/opensourcepos/opensourcepos#faq) for any known install and/or upgrade gotchas (in specific PHP extensions installed)
- [] Read the [wiki](https://github.com/opensourcepos/opensourcepos/wiki)
- [] Executed any database upgrade scripts if an upgrade pre 3.0.0 (e.g. database/2.4_to_3.0.sql)
- [] Aware the installation code is in [bintray](https://bintray.com/jekkos/opensourcepos/opensourcepos/view/files?sort=updated&order=asc#files) (see README), and [GitHub master](https://github.com/opensourcepos/opensourcepos/tree/master) is for [developers only](https://github.com/opensourcepos/opensourcepos/wiki/Development-setup) and therefore not complete nor stable

### Installation information

- OSPOS version is: 
- OSPOS git commit hash is: 
- PHP version is: 
- MySQL or MariaDB version is: 
- OS and version is: 
- WebServer is: 
- Selected language is: 
- (If applicable) Docker installation: 
- (If applicable) Installation package for the LAMP/LEMP stack is: 

### Issue / Bug  / Question / New Feature

Hi!

Can one change barcode layout wording from retail pricing to pricing?

**Comment by data2base** [comment]
![Screenshot_1](https://user-images.githubusercontent.com/37272312/67555122-feed1b00-f710-11e9-88a5-f09c265d2545.jpg)

**Comment by WebShells** [comment]
@data2base actually the titles were removed a while ago as i remember, if you mean the "retail price:" printed word next to the price on barcode.

**Comment by data2base** [comment]
Yes please i would like to change the wording from retail price to price on the barcode.

**Comment by WebShells** [comment]
Please go application, libraries, barcode_lib.php, before the end of the file you can find it...

**Comment by data2base** [comment]
elseif($layout_type == 'unit_price' && isset($item['unit_price']))
		{
			$result = $this->CI->lang->line('items_unit_price') . " " . to_currency($item['unit_price']);

**Comment by data2base** [comment]
i can find this but nowhere the wording, retail price?

**Comment by WebShells** [comment]
You just found the right lines... 
$this->CI->lang->line('items_unit_price') . " " .
Either remove this, or edit items_unit_price in language files to empty string.

**Comment by WebShells** [comment]
Please next time fill in template issue, shall we close?
